sw/device/tests/aon_timer_wdog_lc_escalate_test.c

```diff
@@ -37,8 +37,13 @@ enum {
   kWdogBarkMicros = 3 * 1000,          // 3 ms
   kWdogBiteMicros = 4 * 1000,          // 4 ms
   kEscalationPhase0Micros = 1 * 1000,  // 1 ms
-  kEscalationPhase1Micros = 5 * 1000,  // 5 ms
-  kEscalationPhase2Micros = 500,       // 500 us
+  // The cpu value is slightly larger as the busy_spin_micros
+  // routine cycle count comes out slightly smaller due to the
+  // fact that it does not divide by exactly 1M
+  // see sw/device/lib/runtime/hart.c
+  kEscalationPhase0MicrosCpu = kEscalationPhase0Micros + 200,  // 1.2 ms
```

Comment:
let me know if you think should be a more "accurate" value.
What's happening right now is that 1ms gets divided into 1.04M instead of 1M in `busy_spin_macro`. 
So the cycle count comes out a little smaller, and this causes a mismatch in the test later on. 

It could be better to just make busy_spin_micro more accurate, since it only needs to do the division once.